sips/sip-03x/sip-03x-btc-addresses-for-stacks-transactions.md

```diff
@@ -0,0 +1,215 @@
+* Wallets initiating sBTC sends could check for this parameter. If absent, they should display a warning to the user, informing  them that the recipient's wallet might not recognize the sBTC transaction, even if the underlying Bitcoin address is valid.
+
+### **Principals in smart contracts**
+
+Principals in smart contracts will, in the absence of a future SIP (see “Future Work”), continue to use the Stacks Address format. As such, developer tools, applications and wallets should convert Bitcoin addresses specified as principals for smart contracts to their corresponding Stacks Address, when appropriate.
+
+### **Derivation paths**
+
+Wallets supporting both layer 1 bitcoin and Stacks transactions SHOULD continue to use the BIP-44 cointype (5757) for Stacks when generating the derivation path for addresses. Wallets should provide support for checking the layer 1 bitcoin BIP-44 cointype (0) for Stacks assets and sweeping them to the equivalent Stacks BIP-44 cointype.
+
+## **Related Work**
+
+### **Bitcoin Address Formats**
+
+This proposal leverages existing Bitcoin address standards, specifically P2PKH (Pay-to-Public-Key-Hash) and P2WPKH (Pay-to-Witness-Public-Key-Hash).
+
+### **Lightning Network**
+
+The Lightning Network uses its own invoice format (BOLT11), which is different from on-chain Bitcoin addresses. This SIP aims to avoid introducing a new, distinct standard for sBTC addresses, learning from the friction this can cause.
+
+### **Multi-network Asset Support on Exchanges**
+
+Many cryptocurrency exchanges already support transferring the same asset (e.g., USDT, USDC) across multiple blockchain networks, often using the native address format of each respective network. This demonstrates a precedent for users managing assets at the same conceptual "address" (derived from the same key) but on different networks. This SIP proposes a similar user experience for Bitcoin and sBTC.
+
+## **Backwards Compatibility**
+
+This proposal is designed to be backwards compatible at the consensus layer, as no changes to the Stacks blockchain protocol itself are required.
+
+* Existing Stacks addresses will continue to function.  
+* The changes primarily affect client-side software (wallets, libraries, dApps).  
+* Applications can choose to implement support for Bitcoin addresses gradually.  
+* Users who prefer to use the existing Stacks address format can continue to do so.  
+* Smart contracts and on-chain logic will continue to use the underlying Stacks address format. The conversion to/from Bitcoin addresses will happen at the application/library layer before interacting with contracts.
+
+## **Activation**
+
+This SIP will be considered activated once the following conditions are met:
+
+1. **Updated Libraries**: Key Stacks developer libraries (e.g., Stacks.js or its successors) are updated to:  
```

Comment:
We should probably commit to a list of these, so there can be a checklist for activation.